content/en/docs/reference/config/security/authorization-policy/index.html

```diff
@@ -24,9 +24,19 @@
 <li>Deny the request.</li>
 </ol>
 
-<p>For example, the following authorization policy sets the <code>action</code> to &ldquo;ALLOW&rdquo;
-to create an allow policy. The default action is &ldquo;ALLOW&rdquo; but it is useful
-to be explicit in the policy.</p>
+<p>Istio Authorization Policy also supports the AUDIT action to decide whether to log requests.
+AUDIT policies do not affect whether requests are allowed or denied to the workload.
+Requests will be allowed or denied based solely on ALLOW and DENY policies.</p>
+
+<p>A request will be internally marked that it should be audited if there is an AUDIT policy on the workload that matches the request.
+A separate plugin must be configured and enabled to actually fulfill the audit decision and complete the audit behavior.
+The request will not be audited if there are no such supporting plugins enabled.
+Currently, the only supported plugin is the <a href="/docs/reference/config/proxy_extensions/stackdriver/">Telemetry v2 Stackdriver</a> plugin.</p>
```

Comment:
I'm not sure where this came from but I'd like to remove the `Telemetry v2` before `Stackdriver`

Reply:
This comes from istio.io/api. https://github.com/istio/api/pull/1618 created to remove.